ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -263,12 +270,15 @@ impl TraceBuilder {
         // Decide how to translate each AOT instruction.
         for (iidx, inst) in blk.insts.iter().enumerate() {
             match inst {
-                aot_ir::Inst::Br { .. } => Ok(()),
+                aot_ir::Inst::Br { .. } => Ok(None::<Option<BBlockId>>),
                 aot_ir::Inst::Load {
                     ptr,
                     tyidx,
                     volatile,
-                } => self.handle_load(bid, iidx, ptr, tyidx, *volatile),
+                } => {
+                    self.handle_load(bid, iidx, ptr, tyidx, *volatile)?;
+                    Ok(None)
```

Comment:
Would I be an idiot if I said that these `Ok(None)`s are not assigned to a variable and can thus be removed?